plugins/pasteurl/trumbowyg.pasteurl.js

```diff
@@ -0,0 +1,61 @@
+/* ===========================================================
+ * trumbowyg.pasteurl.js v1.0
+ * Url paste handling with noembed. Plugin for Trumbowyg
+ * http://alex-d.github.com/Trumbowyg
+ * ===========================================================
+ * Author : Max Seelig
+ *          Facebook : https://facebook.com/maxse
+ *          Website : https://www.maxmade.nl/
+ */
+
+(function ($) {
+    'use strict';
+
+    $.extend(true, $.trumbowyg, {
+        plugins: {
+            pasteImage: {
+                init: function (trumbowyg) {
+                    trumbowyg.pasteHandlers.push(function (pasteEvent) {
+                        try {
+                          var clipboardData = (pasteEvent.originalEvent || pasteEvent).clipboardData;
+                          var pastedData = clipboardData.getData('Text');
+                          var request = null;
+                              if(pastedData.startsWith("http")){
+                                pasteEvent.stopPropagation();
+                                pasteEvent.preventDefault();
+
+                                var url = pastedData;
+                                var query = { url : url.trim() };
+
+                                if (request && request.transport) request.transport.abort();
+
+                                request = $.ajax({
+                                  crossOrigin: true,
+                                  url: "https://noembed.com/embed?nowrap=on",
+                                  type: "GET",
+                                  data: query,
+                                  cache: false,
+                                  dataType: "json",
+                                  success: function(res) {
+                                    if (res.html) {
+                                      let content = res.html;
+                                      trumbowyg.execCmd('insertHTML', content);
+                                    }
+                                    else {
+                                      let content = "<a href='"+pastedData+"'>"+pastedData+"</a>";
```

Comment:
```javascript
content = $('<a>', {
	href: pastedData,
	text: pastedData
})
```

Reply:
Isn't <code>let</code> standard ES6?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
Yep but Trumbowyg is ES5 for moment.

Reply:
I don't understand this:
<code>content = $('<a>', {
	href: pastedData,
	text: pastedData
})</code>
How do I get a string out of it like
![image](https://user-images.githubusercontent.com/7907436/36172542-8a3ea590-1106-11e8-89d5-767f4152b649.png)
?

Reply:
You right.

```javascript
content = $('<a>', {
	href: pastedData,
	text: pastedData
}).prop('outerHTML')
```